.github/workflows/lint-go.yaml

```diff
@@ -23,5 +23,5 @@ jobs:
           version: v1.60.3
           # use the default if on main branch, otherwise use the pull request config
           args: --timeout=30m --config=.golangci.yml
-          only-new-issues: true
+          only-new-issues: false
```

Comment:
```suggestion
          only-new-issues: ${{ github.event_name == 'pull_request' }}
```

Reply:
This seems to assume that it's expected only to detect new issues in pull requests, which I believe may fail in certain edge cases like the scenario that brought us here.
While there's a benefit of only running the lint for the diffs, golangci-lint linter is so fast that the difference looks negligible ?